gnovm/tests/file.go

```diff
@@ -124,7 +126,9 @@ func RunFileTest(rootDir string, path string, opts ...RunFileTestOption) error {
 	store := TestStore(rootDir, "./files", stdin, stdout, stderr, mode)
 	store.SetLogStoreOps(true)
 	m := testMachineCustom(store, pkgPath, stdout, maxAlloc, send)
-
+	// set a gasMeter for machine that runs the tests, consider the limit of this
```

Comment:
```suggestion
	// Set a gas meter for machine that runs the tests.
```
"consider the limit of this" is a bit confusing; this is the purpose of defining the gas meter so no need to state this again.